validator/keymanager/v2/direct/import.go

```diff
@@ -39,6 +39,14 @@ func (dr *Keymanager) ImportKeystores(
 			return errors.Wrap(err, "could not add to progress bar")
 		}
 	}
+	foundKey := map[string]bool{}
+	for i := range pubKeys {
+		strKey := hex.EncodeToString(pubKeys[i])
```

Comment:
Nit: no need to encode to a hex string. This would only slow down this method. You can simply cast a byte slice to string and that will work fine as a map key.

```suggestion
		strKey := string(pubKeys[i])
```